smart_dog_app/src/main.js

```diff
@@ -9,7 +9,6 @@ import ProfilePage from "./containers/ProfilePage";
 
 const Main = () => (
   <Switch>
-    W
```

Comment:
Why is there a 'W'??

Reply:
You gotta fix that before I pull it through. If you push a new commit to your branch it auto updates the pull request.